### PR TITLE
OCPBUGS#3629: Add a note regarding enabling SCP in AWS account

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-3629

Link to docs preview: https://56186--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-configuration-parameters-optional_installing-aws-customizations

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -677,6 +677,10 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
+[NOTE]
+====
+If your AWS account have SCP (Service control policies) enabled, you must modify `install-config.yaml` file and add the `credentialsMode: Mint` entry.
```
Comment (yunjiang29) [inline modules/installation-configuration-parameters.adoc:682]: @xenolinux all of `Mint`, `Passthrough` and `Manual` work with this situation.

cc @jianping-shu @huangmingxia for review.
cc @patrickdillon to be aware of this change.

Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc:682]: @yunjiang29  If the AWS account has SCP enabled, then do we have to enable all of these 3 entries? or do we only have to enable the `credentialsMode: Mint` entry?

Comment (yunjiang29) [reply modules/installation-configuration-parameters.adoc]: @xenolinux  only one mode can be specified in install-config: `Mint` or `Passthrough` or `Manual`

Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc]: Addressed

**Comment by huangmingxia** [comment]
LGTM.

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -677,6 +677,10 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
+[NOTE]
+====
+If your AWS account has SCP (Service control policies) enabled, you must modify `install-config.yaml` file and add the `credentialsMode: Mint`, `credentialsMode: Passthrough` or `credentialsMode: Manual` entry.
+====
 |`Mint`, `Passthrough`, `Manual`, or an empty string (`""`).
```
Comment (jianping-shu) [inline modules/installation-configuration-parameters.adoc:684]: @xenolinux What does `""` mean here? 
It means no entry for credentialsMode or "credentialsMode: "?

Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc:684]: I guess it means no entry for the `credentialsMode`.

Comment (yunjiang29) [reply modules/installation-configuration-parameters.adoc:684]: `credentialsMode: ""` equals _NO `credentialsMode` entry in install-config_, it's the default mode.

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -677,6 +677,10 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
+[NOTE]
+====
+If your AWS account has SCP (Service control policies) enabled, you must modify `install-config.yaml` file and add the `credentialsMode: Mint`, `credentialsMode: Passthrough` or `credentialsMode: Manual` entry.
```
Comment (yunjiang29) [inline modules/installation-configuration-parameters.adoc:682]: How about change the note like:
> If your AWS account has SCP (Service control policies) enabled, `credentialsMode` must be configured to `Mint`, `Passthrough` or `Manual` explicitly.

Comment (jianping-shu) [reply modules/installation-configuration-parameters.adoc:682]: + in the install-config.yaml file.

File: modules/installation-configuration-parameters.adoc (hunk 2 above)
Comment (jianping-shu) [reply modules/installation-configuration-parameters.adoc:684]: @yunjiang29 Did you ever run credentialsMode: "" for SCP enable case? We never tested it but Mingxia can have a try.

Comment (jianping-shu) [reply modules/installation-configuration-parameters.adoc:684]: Mingxia just had a try. Yes, credentialsMode: "" equals to no entry so the install failed when SCP is enabled.
@xenolinux We need to delete the part of ', or an empty string (`""`).'

Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc]: @jianping-shu Addressed- removed the part "or an empty string ("")".

File: modules/installation-configuration-parameters.adoc (hunk 3 above)
Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc]: @yunjiang29 Addressed

File: modules/installation-configuration-parameters.adoc (hunk 2 above)
Comment (jianping-shu) [reply modules/installation-configuration-parameters.adoc]: @xenolinux Pls. roll back this change per my comment. This value list should be the full list and not related to SCP issue.

**Comment by jianping-shu** [comment]
@xenolinux Sorry. I didn't read the full context in the document page so my comments are misleading. Pls. roll back the changes for my 2 comments, the rewording in @yunjiang29 's comment looks good to me.
Thanks

File: modules/installation-configuration-parameters.adoc (hunk 3 above)
Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc]: Addressed

**Comment by jianping-shu** [comment]
@xenolinux  I asked the roll back for 2 places. Seems you didn't restore back ', or an empty string (`""`).'

**Comment by xenolinux** [comment]
> @xenolinux I asked the roll back for 2 places. Seems you didn't restore back ', or an empty string (`""`).'

@jianping-shu Addressed

**Comment by jianping-shu** [comment]
LGTM

File: modules/installation-configuration-parameters.adoc [hunk 4]
```diff
@@ -677,7 +677,11 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
-|`Mint`, `Passthrough`, `Manual`, or an empty string (`""`).
+[NOTE]
+====
+If your AWS account has SCP (Service control policies) enabled, you must configure the `credentialsMode` parameter to `Mint`, `Passthrough` or `Manual`.
```
Comment (mburke5678) [inline modules/installation-configuration-parameters.adoc:682]: I think you should s/SCP (Service control policies)/service control policies (SCP)

Note the lower-case `s` in `service`.
See: https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_inheritance_auth.html

**Comment by mburke5678** [comment]
@xenolinux One tiny nit. Otherwise LGTM

File: modules/installation-configuration-parameters.adoc (hunk 4 above)
Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc]: Addressed

**Pull request merged**